packages/tocco-util/src/color/contrast.js

```diff
@@ -0,0 +1,25 @@
+const LUMIANCE_LIMIT = 0.179
+
+const calculateLight = colorItem => {
+  let c = colorItem / 255.0
+  if (c <= 0.03928) {
+    c = c / 12.92
+  } else {
+    c = Math.pow((c + 0.055) / 1.055, 2.4)
+  }
+  return c
+}
+
+const calculateLumiance = color =>
+  0.2126 * calculateLight(color.r) + 0.7152 * calculateLight(color.g) + 0.0722 * calculateLight(color.b)
+
+/***
+ * Returns a dark or bright value depending on the given color.
+ * This could be used to choose a pleasant font color to a background color.
+ * @param rgbColor Object whit this shape: {r: 0, g: 0, b: 0}
+ * @param brightValue Bright value can be overridden. Default is the color white as hex.
+ * @param darkValue Dark value can be overridden. Default is the color black as hex.
+ * @returns {string}
+ */
+export const getContrastColor = (rgbColor, brightValue = '#FFFFFF', darkValue = '#000000') =>
```

Comment:
I personally don't like that we use two different formats for parameters of the same type. `rgbColor`, `brightValue` and `darkValue` all represent a color, but for `rgbColor` we use an object and for the other two a string and we also return a string.

What about expecting for `rgbColor` a hex string as well and parse it here instead of in the scheduler? I reckon 
 this util would be more useful in other places as well if it could handle a hex string for `rgbColor`.